### DISCUSSION TITLE
Can anyone start a node to join in crossbell network?

### DISCUSSION THREAD

**Discussion by litrane** [Crossbell-Box/Crossbell-Contracts#98] [Q&A] [answered]
I find that the source code of corssbell network node is not open.

**Comment by Atlasoin** [comment, marked as answer]
Hi, currently it's not open sourced but has been scheduled. And our validator nodes all require approval. Starting a node to join is not open yet. You can learn more here: https://docs.crossbell.io/docs/concepts/architecture#chain-and-contracts

Thanks for your interest and feel free to leave any of your questions if there's any.

**Reply by litrane** [reply]
Got it. Thanks!

**Comment by dxhuii** [comment]
期待早日开源
